src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final Random random = new Random();
+
+    public Ball getRandomBall() {
+        int upperLimit = 100;
```

Comment:
here is how constant looks like: private static final int MAX_RANDOM_NUMBER = 100;